atomics/T1010/T1010.yaml

```diff
@@ -11,18 +11,33 @@ atomic_tests:
     - windows
 
   input_arguments:
+    input_url:
+      description: URL to source code in Atomic-Red-Team git repository
+      type: Url
+      default: https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/atomics/T1010/src/T1010.cs
     input_source_code:
       description: Path to source of C# code
```

Comment:
Since the cs file is a prereq, could you put it right into the atomics folder instead of tmp